CMakeLists.txt

```diff
@@ -100,4 +162,5 @@ foreach(h ${PUBLIC_HEADERS})
         COMPONENT "headers")
 endforeach()
 
-add_subdirectory(examples)
+
+# add_subdirectory(examples)
```

Comment:
Put it under condition instead of commenting out:
```cmake
OPTION(AUTOBAHN_CPP_BUILD_EXAMPLES "Build examples" OFF)
if(AUTOBAHN_CPP_BUILD_EXAMPLES)
  add_subdirectory(examples)
endif()
```